src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -40,3 +40,6 @@ dataverse.oai.server.maxsets=100
 # the OAI repository name, as shown by the Identify verb,
 # can be customized via the setting below:
 #dataverse.oai.server.repositoryname=
+
+# METADATA SETTINGS
+#dataverse.metadata.block-system-metadata-keys
```

Comment:
Not sure why you add this here, IMHO we should only add things that we have a default for. (Yes I am aware that there is a similar thing for OAI above - yet it looks like people start to mistake this file as editable for configuration, which is not how it's meant to be used)